ecosystem/indexer-grpc/indexer-grpc-cache-worker/src/worker.rs

```diff
@@ -255,6 +278,10 @@ async fn process_streaming_response(
                     current_version += num_of_transactions;
                     transaction_count += num_of_transactions;
                     tps_calculator.tick_now(num_of_transactions);
+
+                    PROCESSED_VERSIONS_COUNT.inc_by(num_of_transactions);
```

Comment:
If the cache worker has to reprocess some transactions, is it possible for `PROCESSED_VERSIONS_COUNT` to be out of sync with the latest processed version? 

Or do we only care if the number is increasing?